.env

```diff
@@ -43,4 +43,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###< symfony/mailer ###
 ARTICLES_ON_PAGE=10
 PROGRAMME_MIN_TIME_MINUTES=15
-PROGRAMME_MAX_TIME_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_MINUTES=360
```

Comment:
@marceldobrica this PR holds the fixes from this PR: https://github.com/marceldobrica/training/pull/34

Please create specific and isolated PRs.

Reply:
Pease rebase this one